scripts/tools/common.yml

```diff
@@ -125,6 +125,18 @@ etors:
       desc: $t_metric_calculate_exp_desc_t
       version: "1.7"
       value: "0x00010002"
+    - name: METRIC_PROGRAMMABLE_EXP_PROPERTIES
+      desc: $t_metric_programmable_exp_properties_t
+      version: "1.8"
```

Comment:
@joshuaranjan version to 1.9 here as well

Reply:
Sorry missed it.
Updated now.